### DISCUSSION TITLE
Context variable cannot be query object

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#655] [Q&A]
Dear Developers,

I tried to use contents of a context variable as query object (`FROM` term),
```sql
LET $a = (SELECT FROM mytype);
SELECT FROM $a;
```
yet I get the following error returned:
```
Internal error Type with name '$a' was not found
```
 Maybe it is generally illegal to use variables inside `FROM` or could this be a bug?

**Comment by lvca** [comment]
Have you tried with `sqlscript` language?
